example.raml

```diff
@@ -44,6 +44,27 @@ securitySchemes:
                 allow {{user_self}} patch
 securedBy: [x_ticket_auth]
 
+/staff:
+    displayName: All stories
```

Comment:
not sure it matters, but the descriptions are wrong here. It's not stories, but users.